src/plugins/hetero/tests/functional/shared_tests_instances/behavior/ov_plugin/life_time.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/ov_plugin/life_time.hpp"
+
+using namespace ov::test::behavior;
+
+namespace {
+
+INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests,
+                         OVHoldersTestOnImportedNetwork,
+                         ::testing::Values(ov::test::utils::DEVICE_TEMPLATE, "HETERO:TEMPLATE"),
+                         OVHoldersTestOnImportedNetwork::getTestCaseName);
```

Comment:
why only test on import network? how about OVHoldersTest
and also, FYI, from previous experience, lifetime for tensor management should be tested like this:
src/plugins/auto/tests/unit/life_time_test.cpp
because in general, template plugin will not trigger issue, because the tensors are in system memory

Reply:
@songbell Please create a ticket to improve the coverage with CPU/GPU in nightly test